app/services/common/shared_data.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Created by Rajinda Waruna on 25/04/2018.
+ */
+
+agentApp.factory('shared_data', function () {
+    return {agent_statue:"Offline", call_task_registered:false, currentModeOption:"",firstName:"",phone_config:{},callDetails: {},acw_time:10,phone_strategy:"",userProfile : {},userAccessFields : {}};//veery_web_rtc_phone
```

Comment:
typo should be 'agent_status' not 'agent_statue'

Reply:
property name change to agent_status.